pkg/kinflate/commands/inflate.go

```diff
@@ -82,7 +83,8 @@ func (o *inflateOptions) Complete(cmd *cobra.Command, args []string) error {
 // RunKinflate runs inflate command (do real work).
 func (o *inflateOptions) RunKinflate(out, errOut io.Writer) error {
 	// Build a tree of ManifestData.
-	root, err := tree.LoadManifestDataFromPath(o.manifestPath)
+	loader := tree.Loader{FS: fs.MakeRealFS(), Path: o.manifestPath}
+	root, err := loader.LoadManifestDataFromPath(o.manifestPath)
```

Comment:
why does the loader need the path twice?

Reply:
We added a `Path` member to the Loader struct with the aim that it will help us improving the error message. We don't need the second `o.manifestPath`, I will remove it.